### PR TITLE
feat: new invalid asyncapi template 

### PR DESCRIPTION
**Pull request by starlightknown**
**Description**

- added a new asyncapi invalid document for the studio tutorial
- updated with tutorial example in index.tsx , NewFileModal.tsx
- ...

**Related issue(s)**

### CODE REVIEW

File: src/components/Modals/NewFileModal.tsx [hunk 1]
```diff
@@ -52,6 +52,7 @@ export const NewFileModal = create(() => {
 
   const realLifeExamples = examples.filter((template) => template.type === 'real-example');
   const templates = examples.filter((template) => template.type === 'protocol-example');
+  const templates = examples.filter((template) => template.type === 'tutorial-example');
```
Comment (magicmatatjahu) [inline src/components/Modals/NewFileModal.tsx:55]:
```suggestion
  const tutorials = examples.filter((template) => template.type === 'tutorial-example');
```

File: src/examples/index.tsx [hunk 2]
```diff
@@ -6,13 +6,17 @@ import websocket from './websocket-gemini.yml';
 import mqtt from './streetlights-mqtt.yml';
 import simple from './simple.yml';
 
+// tutorial example
+import tutorial from './tutorials/tutorial.yml';
```
Comment (magicmatatjahu) [inline src/examples/index.tsx:10]: In the future we will have more tutorials so it's better to call it as (e.g.) `invalid` to describe purpose of the tutorial.

```suggestion
// tutorial example
import invalid from './tutorials/invalid.yml';
```

File: src/examples/index.tsx [hunk 3]
```diff
@@ -22,6 +26,12 @@ export default [
     template: simple,
     type: templateTypes.protocol
   },
+  {
+    title: 'Tutorial Example',
+    description: () => <>An example of an invalid AsyncAPI document. This is only for educational purposes, to learn document validation.</>,
+    template: tutorial,
+    type: templateTypes.tutorial
```
Comment (magicmatatjahu) [inline src/examples/index.tsx:33]:
```suggestion
    title: 'Invalid Example',
    description: () => <>An example of an invalid AsyncAPI document. This is only for educational purposes, to learn document validation.</>,
    template: invalid,
    type: templateTypes.tutorial
```

File: src/components/Modals/NewFileModal.tsx (hunk 1 above)
Comment (magicmatatjahu) [reply src/components/Modals/NewFileModal.tsx:55]: Also, in this same file you need to update the React component to render tutorials:

```tsx
          <div>
            <span className="uppercase text-gray-800 text-sm underline font-bold">Tutorials</span>
            <div className="grid grid-cols-3 gap-4 py-4">
              {tutorials.map(({ title, description, template }) => {
                const isSelected = selectedTemplate.title === title;
                return <TemplateListItem title={title} description={description} isSelected={isSelected} key={title} onClick={() => setSelectedTemplate({ title, template })} />;
              })}
            </div>
          </div>
```

please paste it under 84 line :)

**Review by magicmatatjahu — CHANGES_REQUESTED** [review]
@starlightknown Thanks for the PR! Awesome, but we need to fix some things :) Let me know if you will have a problems.

**Comment by starlightknown** [comment]
image
yay its on

**Review by magicmatatjahu — APPROVED** [review]

**Review by fmvilas — APPROVED** [review]
🚀

**Comment by magicmatatjahu** [comment]
@fmvilas https://github.com/asyncapi/website/pull/1022#issuecomment-1354032705 XD

**Pull request merged**